modules/runes/api/httphandler/get_transactions.go

```diff
@@ -279,12 +294,12 @@ func (h *HttpHandler) GetTransactions(ctx *fiber.Ctx) (err error) {
 		}
 		txList = append(txList, respTx)
 	}
-	// sort by block height ASC, then index ASC
+	// sort by block height DESC, then index DESC
 	slices.SortFunc(txList, func(t1, t2 transaction) int {
 		if t1.BlockHeight != t2.BlockHeight {
-			return int(t1.BlockHeight - t2.BlockHeight)
+			return int(t2.BlockHeight - t1.BlockHeight)
 		}
-		return int(t1.Index - t2.Index)
+		return int(t2.Index - t1.Index)
```

Comment:
for the good practice, it should be used `cmp.Compare` here to prevent underflow

Reply:
fixed, but there are many more cases where this would apply, will fix in later PR